### ISSUE TITLE
🐛 2.4 fails to parse scss interpolation correctly

### ISSUE THREAD

**Issue by mrdrogdrog** [biomejs/biome#9090]
### Environment information

<details>

```bash

CLI:
  Version:                      2.4.0
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:                    unset
  BIOME_LOG_PREFIX_NAME:             unset
  BIOME_LOG_LEVEL:                   unset
  BIOME_LOG_KIND:                    unset
  BIOME_CONFIG_PATH:                 unset
  BIOME_THREADS:                     unset
  BIOME_WATCHER_KIND:                unset
  BIOME_WATCHER_POLLING_INTERVAL:    unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v24.13.1
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  false
  HTML full support enabled:    unset

Workspace:
  Open Documents:               0


```

</details>

### What happened?

It seems like biome 2.4 fails to parse SCSS interpolation correctly which is necessary for e.g. assigning a scss var to a css var.

Here is an example:
1. Put the following code into a "index.scss" file. (couldnt put it into the playground because it doesnt support scss?)
```
$red: #a5031e;
:root {
        --bs-primary: #{$red};
}
```

2. Run `biome check` on the file. 

Output: 
```
  ✖ Expected a qualified rule, or an at rule but instead found ';
    }'.
  
    1 │ $red: #a5031e;
    2 │ :root {
  > 3 │ 	--bs-primary: #{$red};
      │ 	                    ^
  > 4 │ }
      │ ^
    5 │ 
  
  ℹ Expected a qualified rule, or an at rule here.
  
    1 │ $red: #a5031e;
    2 │ :root {
  > 3 │ 	--bs-primary: #{$red};
      │ 	                    ^
  > 4 │ }
      │ ^
    5 │ 
```
 

### Expected result

The file should be accepted and processed like in biome 2.3

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
Biome doesn't support SCSS, that was probably a bug that got fixed. However biome shouldn't even parse a SCSS file. Can you create a reproduction?

**Comment by mrdrogdrog** [comment]
> Biome doesn't support SCSS

HUH okay. Never noticed that. I just verified that with my example code and yes, biome 2.3 totally ignores the file. I use a biome plugin for my IDE that overrides the IDEs own code formatting and linting so I guess this is why I assumed that biome would check/format my scss code. But apparently the scss files have been processed by webstorms own formatter.

Anyway.

Here is a reproduction. https://github.com/mrdrogdrog/biome-scss-reproduction . This file is ignored by biome 2.3.13 but crashes with 2.4.

**Comment by mrdrogdrog** [comment]
Is scss support planned?

**Comment by ematipico** [comment]
@mrdrogdrog yeah it is, and we started to work on the parsing, however in doing so we introduced this regression :)

**Comment by mrdrogdrog** [comment]
Thank you @ematipico for the quick response. You're awesome!
